### PR TITLE
Added changes for custome headers in Python-SDK for V3

### PR DESCRIPTION
**Pull request by Rrathinasabapath**
Please review the changes for custom headers for V4

### CODE REVIEW

File: hyperwallet/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 __email__           = 'devsupport@hyperwallet.com'
 __copyright__       = 'Copyright (c) 2018 Hyperwallet'
 __license__         = 'MIT'
-__version__         = '1.5.1'
+__version__         = '1.6.0'
```
Comment (wmews-hw) [inline hyperwallet/__init__.py:9]: if this is v4 shouldn't it be 2.x.x???

File: python-sdk.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wmews-hw) [inline python-sdk.iml:1]: Is there a reason to commit this file?

Comment (arao6) [reply python-sdk.iml]: File removed

File: hyperwallet/__init__.py (hunk 1 above)
Comment (arao6) [reply hyperwallet/__init__.py:9]: This is V3 Python. Title is incorrect

**Review by wmews-hw — APPROVED** [review]

**Review by arao6 — APPROVED** [review]

**Pull request merged**